hail/src/main/scala/is/hail/expr/types/physical/PArrayBackedContainer.scala

```diff
@@ -71,6 +71,9 @@ trait PArrayBackedContainer extends PContainer {
   def setElementMissing(region: Region, aoff: Long, i: Int) =
     arrayRep.setElementMissing(region, aoff, i)
 
+  def setElementMissing(aoff: Long, i: Int) =
```

Comment:
This makes the staged and unstaged setElementMissing have constructors with same arity (namely sans region). I can hold off on this change if you prefer, for now.